number_18/index.js

```diff
@@ -0,0 +1,59 @@
+const pythagorean = (a, b) => {
+  const c = Math.sqrt(a * a + b * b);
+  return c;
+};
+console.log(pythagorean(5, 12));
+
+const generatePassword = (n) => {
+  let characters =
+    "!\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz";
+  let password = "";
+  for (let i = 0; i < n; i++) {
+    let randomIndex = Math.floor(Math.random() * characters.length);
+    password += characters.charAt(randomIndex);
+  }
+  return password;
+};
+console.log(generatePassword(8));
+
+const isPrime = (n) => {
+  let kv = Math.sqrt(n)
```

Comment:
Why do you need kv?

Reply:
I was in a hurry, so I called the root square